### DISCUSSION TITLE
Test cases for shiny NCA dashboard

### DISCUSSION THREAD

**Discussion by Ravua1992** [humanpred/pknca#212] [General]
I really enjoy using PKNCA and I use it on a daily basis either while simulating via mrgsolve or other packages.  
I created a small dashboard allowing users to take input CSV files and perform NCA and exploratory graphical analysis by selecting various columns and providing inputs.
Can you please tell me the best test cases to ensure the dashboard is running as expected? 

Thank you! 

link to the dashboard: 

https://pmxvidhya.com/flexnca/

**Comment by billdenney** [comment]
@Ravua1992, thanks for your interest!  I've converted this from an issue to a discussion.

It sounds like you're looking for operational qualification (OQ) or performance qualification (PQ) for the dashboard.  I think that the most straight-forward test case is to use the theophylline example from the theophylline vignette: http://billdenney.github.io/pknca/articles/v02-example-theophylline.html

**Reply by Ravua1992** [reply]
Thank you, Bill.
